cassandra/src/cassandra/nemesis.clj

```diff
@@ -60,8 +60,12 @@
                               (str "nemesis already disrupting " @nodes))
                             :no-target)
                    :stop (if-let [ns @nodes]
-                           (let [restarted (for [node ns] (c/on node (stop! test node)))]
-                            (reset! nodes nil)
+                           (let [all-nodes (:nodes test)
+                                 seed (first (cass/seed-nodes test))
+                                 all-crashed? (= (count ns) (count all-nodes))
+                                 reordered (if all-crashed? (conj (remove #(= seed %) ns) seed) ns)
```

Comment:
Just a question: suppose 5 nodes, rf = 3, and 2 seed nodes. If the nemesis crashes three nodes including the 2 seeds will there be a problem if Jepsen tries to restart the non-seed node first?

I think that in this case the alive nodes know how to find the seed already once it comes up. How about the case of the restarted node? I suppose it will have to wait until one of the seed nodes is up before it can rejoin the cluster, but that should be pretty quick so no problem?

Reply:
C* node can restart if at least one node is alive whether the living node is a seed or not.

Reply:
Ah, I see. Thank you!